pkg/kubelet/volumemanager/cache/actual_state_of_world_test.go

```diff
@@ -192,6 +192,9 @@ func Test_AddPodToVolume_Positive_ExistingVolumeNewNode(t *testing.T) {
 	}
 	volumeSpec := &volume.Spec{Volume: &pod.Spec.Volumes[0]}
 	generatedVolumeName, err := volumehelper.GetUniqueVolumeNameFromSpec(plugin, volumeSpec)
+	if err != nil {
```

Comment:
Use testify assertions for these? e.g. https://godoc.org/github.com/stretchr/testify/assert#NoError
I found out that we vendor this library the other day, it's quite nice.

Reply:
@mtaufen It's so cool, so that i can't stop myself to modify all appears in this file, ^_^, PTAL, thank you!